<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKBroadcastAssistant</class>
 <widget class="QDialog" name="SAKBroadcastAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Broadcast parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxBroadcastAddress"/>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QComboBox" name="comboBoxBroadcastPrefix"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Broadcast port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Broadcast address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="comboBoxBroadcastInterval"/>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interval</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Broadcast data</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditBroadcastPort"/>
      </item>
      <item row="0" column="9">
       <widget class="QComboBox" name="comboBoxBroadcastsuffix"/>
      </item>
      <item row="0" column="12">
       <widget class="QComboBox" name="comboBoxBroadcastFormat"/>
      </item>
      <item row="1" column="1" colspan="11">
       <widget class="QLineEdit" name="lineEditBroadcastData"/>
      </item>
      <item row="1" column="12">
       <widget class="QPushButton" name="pushButtonBroadcast">
        <property name="text">
         <string>Broadcast</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Information output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTextBrowser" name="textBrowserInformation"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>textBrowserInformation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
